firestore/app/src/main/java/com/google/example/firestore/DocSnippets.java

```diff
@@ -1361,4 +1361,23 @@ public void onComplete(@NonNull Task<Void> task) {}
         });
         // [END update_delete_field]
     }
+
+    public void countAggregateQuery() {
+        // [START count_aggregate_query]
+        Query query = db.collection("games/chess/players").whereEqualTo("online", true);
+        AggregateQuery countQuery = query.count();
+        countQuery.get(AggregateSource.SERVER).addOnCompleteListener(new OnCompleteListener<AggregateQuerySnapshot>() {
+    @Override
+    public void onComplete(@NonNull Task<AggregateQuerySnapshot> task) {
+        if (task.isSuccessful()) {
+            // Count fetched successfully
+            AggregateQuerySnapshot snapshot = task.getResult();
+            Log.d(TAG, "Count: " + snapshot.getCount());
+        } else {
+            Log.d(TAG, "Count failed: ", task.getException());
+        }
+    }
+});;
+        // [END count_aggregate_query]
```

Comment:
Is END one or two lines too low? In docs, this would start at line 1367 Query query = db.collection.... Possibly belongs right after line 1378?

Reply:
The indents were a bit funky, they're fixed now.